app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -57,8 +60,10 @@ import java.util.Locale
  * The V in MVP
  * Created by chiragw15 on 9/7/17.
  */
-class ChatActivity : AppCompatActivity(), IChatView {
+class ChatActivity : AppCompatActivity(), IChatView, GestureDetector.OnGestureListener,
```

Comment:
Don't implement the interfaces on the class, use anonymous classes and lambdas

Reply:
Can you please explain how can I write it?